src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -267,7 +299,16 @@ class FilterPopup extends React.Component {
                   onFilterClick={this.onFilterClick}
                 />
 
-                <FieldsetDate legend="Date" onDateFilterChange={this.onDateFilterChange} />
+                <FieldsetDate
+                  legend="Date"
+                  selectedFilters={
```

Comment:
I like making these objects into variables so we only pass variables as props instead of the whole object. It looks cleaner to do `selectedFilters={selectedFilters}`.